### PR TITLE
Install file fixes: MySQL Warning removed

### PR DESCRIPTION
**Pull request by satyammittal**
MySQL Warning removed : leads easily login into mysql without any issues

### CODE REVIEW

File: install/install.sh [hunk 1]
```diff
@@ -43,13 +43,16 @@ echo "-------------------------------"
 echo ""
 echo "In order to configure the platform, we need some information from you. Please reply to the following questions:"
 echo ""
-read -e -p "Password of the 'root' user of MySQL: " -i "" db_root_password
+
 # Verify password
-while ! mysql -u root --password="${db_root_password}"  -e ";" ; do
-      read -e -p "Invalid password, please retry: " -i "" db_root_password
+mysql_config_editor set --login-path=root_login --host=localhost --user=root --password $db_root_password
```
Comment (canihavesomecoffee) [inline install/install.sh]: Where is $db_root_password coming from if I'd merge your PR?

File: install/install.sh [hunk 2]
```diff
@@ -43,13 +43,16 @@ echo "-------------------------------"
...
+as=2;
```
Comment (canihavesomecoffee) [inline install/install.sh]: What's the use of this?

File: install/install.sh [hunk 3]
```diff
@@ -43,13 +43,16 @@ echo "-------------------------------"
...
+echo "checkout\n";
```
Comment (canihavesomecoffee) [inline install/install.sh]: No need for this

File: install/install.sh [hunk 4]
```diff
@@ -43,13 +43,16 @@ echo "-------------------------------"
...
+while ! mysql  --login-path=root_login  -e ";" ; do
+      echo "sasasa\n";
```
Comment (canihavesomecoffee) [inline install/install.sh]: Nor this

File: install/install.sh [hunk 5]
```diff
@@ -43,13 +43,16 @@ echo "-------------------------------"
...
+      supress_warning=`mysql_config_editor set --login-path=root_login --host=localhost --user=root --password $db_root_password` >> "$install_log" 2>&1
```
Comment (canihavesomecoffee) [inline install/install.sh]: Look at your code and see what happens if I enter a wrong password.

File: install/install.sh [hunk 6]
```diff
@@ -84,7 +87,7 @@ supress_warning=`mysql_config_editor set --login-path=user_login --host=localhos
 # Grant user access to database
 mysql -u root --password="${db_root_password}" -e "GRANT ALL ON ${db_name}.* TO '${db_user}'@'localhost';" >> "$install_log" 2>&1
 # Check if user has access
-db_access=`mysql --login-path=user_login -se"USE ${db_name};" 2>&1`
+db_access=`mysql --login-path=root_login -se"USE ${db_name};" 2>&1`
```
Comment (canihavesomecoffee) [inline install/install.sh]: Why did you change this?

Comment (satyammittal) [inline install/install.sh]: error showing "unable to grant access to user for database"

File: install/install.sh (hunk 4 above)
Comment (satyammittal) [inline install/install.sh]: ok i will remove it

File: install/install.sh (hunk 6 above)
Comment (canihavesomecoffee) [inline install/install.sh]: Then that should be fixed instead of using the root user for the database.

Comment (satyammittal) [inline install/install.sh]: ok let me fix it

File: install/install.sh [hunk 7]
```diff
@@ -45,12 +45,9 @@ echo "In order to configure the platform, we need some information from you. Ple
 echo ""
 
 # Verify password
-mysql_config_editor set --login-path=root_login --host=localhost --user=root --password $db_root_password
-as=2;
-echo "checkout\n";
+mysql_config_editor set --login-path=root_login --host=localhost --user=root --password
```
Comment (canihavesomecoffee) [inline install/install.sh]: Removing the password is no solution either.

File: install/install.sh [hunk 8]
```diff
@@ -45,11 +45,12 @@ echo "In order to configure the platform, we need some information from you. Ple
 echo ""
 read -e -p "Password of the 'root' user of MySQL: " -i "" db_root_password
 # Verify password
-while ! mysql -u root --password="${db_root_password}"  -e ";" ; do
-      read -e -p "Invalid password, please retry: " -i "" db_root_password
+$db_root_password
```
Comment (canihavesomecoffee) [inline install/install.sh]: What's this doing here?

File: install/install.sh [hunk 9]
```diff
@@ -45,11 +45,12 @@ echo "In order to configure the platform, we need some information from you. Ple
...
+mysql_config_editor set --login-path=root_login --host=localhost --user=root --password $db_root_password 
+while ! mysql  --login-path=root_login  -e ";" ; do
+      mysql_config_editor set --login-path=root_login --host=localhost --user=root --password $db_root_password
```
Comment (canihavesomecoffee) [inline install/install.sh]: My question still remains: What if the initial password is entered wrongly?

File: install/install.sh [hunk 10]
```diff
@@ -45,11 +45,12 @@ echo "In order to configure the platform, we need some information from you. Ple
 echo ""
 read -e -p "Password of the 'root' user of MySQL: " -i "" db_root_password
 # Verify password
-while ! mysql -u root --password="${db_root_password}"  -e ";" ; do
-      read -e -p "Invalid password, please retry: " -i "" db_root_password
+$db_root_password
+mysql_config_editor set --login-path=root_login --host=localhost --user=root --password $db_root_password
```
Comment (canihavesomecoffee) [inline install/install.sh]: No logging?

File: install/install.sh [hunk 11]
```diff
@@ -80,11 +81,14 @@ else
        read -e -p "Invalid password, please retry: " -i "" db_user_password
     done
 fi
-supress_warning=`mysql_config_editor set --login-path=user_login --host=localhost --user=$db_user --password $db_user_password` >>  "$install_log" 2>&1
+#e=`mysql_config_editor set --login-path=mypath --host=localhost --user=$db_user --password '${db_user_password}' >> "$install_log" 2>&1`
+#echo $e;
+mysql_config_editor set --login-path=mypath --host=localhost --user=$db_user --password ;
```
Comment (canihavesomecoffee) [inline install/install.sh]: Connecting without user password is NOT ok.

Comment (satyammittal) [inline install/install.sh]: it will ask user ti reenter the password on the terminal through stdin

Comment (satyammittal) [inline install/install.sh]: mysql_config_editor is not taking $db_user_password`as argument after --password

Comment (canihavesomecoffee) [inline install/install.sh]: And store that newly entered password where exactly? It's needed further in the install script.

File: install/install.sh (hunk 10 above)
Comment (satyammittal) [inline install/install.sh]: i will add logging.

File: install/install.sh (hunk 8 above)
Comment (satyammittal) [inline install/install.sh]: will remove it..

File: install/install.sh [hunk 12]
```diff
@@ -80,11 +80,13 @@ else
        read -e -p "Invalid password, please retry: " -i "" db_user_password
     done
 fi
-supress_warning=`mysql_config_editor set --login-path=user_login --host=localhost --user=$db_user --password $db_user_password` >>  "$install_log" 2>&1
+#login with user account in mysql by veryfying mysql password
+mysql_config_editor set --login-path=mypath --host=localhost --user=$db_user --password ;
```
Comment (canihavesomecoffee) [inline install/install.sh]: Asking the user again for a password he possibly didn't create (see the section above) is asking for trouble.

File: install/install.sh [hunk 13]
```diff
@@ -80,11 +80,12 @@ else
        read -e -p "Invalid password, please retry: " -i "" db_user_password
     done
 fi
-supress_warning=`mysql_config_editor set --login-path=user_login --host=localhost --user=$db_user --password $db_user_password` >>  "$install_log" 2>&1
+supress_warning=`mysql_config_editor set --login-path=user_login --host=localhost --user=$db_user --password $db_user_password` >> "$install_log" 2>&1
 # Grant user access to database
-mysql -u root --password="${db_root_password}" -e "GRANT ALL ON ${db_name}.* TO '${db_user}'@'localhost';" >> "$install_log" 2>&1
+mysql --login-path=root_login -e "GRANT ALL ON ${db_name}.* TO '${db_user}'@localhost;"
```
Comment (canihavesomecoffee) [inline install/install.sh]: Logging is gone here, please re-add

File: install/install.sh [hunk 14]
```diff
@@ -80,11 +80,12 @@ else
...
 # Check if user has access
-db_access=`mysql --login-path=user_login -se"USE ${db_name};" 2>&1`
+
```
Comment (canihavesomecoffee) [inline install/install.sh]: Please remove this empty line.

File: install/install.sh [hunk 15]
```diff
@@ -44,12 +44,12 @@ echo ""
 echo "In order to configure the platform, we need some information from you. Please reply to the following questions:"
 echo ""
 read -e -p "Password of the 'root' user of MySQL: " -i "" db_root_password
-# Verify password
```
Comment (canihavesomecoffee) [inline install/install.sh]: Please re-add this comment

File: install/install.sh [hunk 16]
```diff
@@ -80,11 +80,12 @@ else
        read -e -p "Invalid password, please retry: " -i "" db_user_password
```
Comment (canihavesomecoffee) [inline install/install.sh]: This case (existing user) still will throw warnings, please add some code to mitigate it here as well.
